### ISSUE TITLE
[A11y] Narrate attachment content inside live region

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3521]
## Version

`4.10.1`

## Describe the bug

Today, for live region, we narrate:

"Bot said, Hi I am Clara! One attachment."

Tomorrow, we want to narrate:

"Bot said, Hi I am Clara! I agree button. I don't agree button."

Also true for card with dropdowns.

## Steps to reproduce

1. Go to MockBot
1. Turn on Windows Narrator
1. Type "herocard qna 1"

## Expected behavior

The narration should be:

"Bot said, Hi I am Clara! I agree button. I don't agree button."

## Additional context

We need to add update a few things

- Update activity middleware to support custom rendering of "live region version" of the activity
   - The middleware must provide a custom render of the "interactive version", which is same as today
   - The middleware is optional to provide a custom render of the "live region version"
- Upgrade all default activity middleware to custom render the live region version, for example
   - Adaptive Cards: retrieve all widgets and custom render it in the live region version
   - Image/animation: instead of saying "one attachment", use the alt text
   - Audio/video: say, "one audio" or "one video"
   - Markdown: say, "rich text"
   - Plain: say, "plain text"
   - File upload/download: say, "file abc.txt of size 12 kB"

[Bug]

**Comment by compulim** [comment]
Done.
